qiskit_dynamics/signals/signals.py

```diff
@@ -24,13 +24,10 @@
 import numpy as np
 from matplotlib import pyplot as plt
 
-try:
-    import jax.numpy as jnp
-except ImportError:
-    pass
-
 from qiskit import QiskitError
-from qiskit_dynamics.array import Array
+from qiskit_dynamics.arraylias import ArrayLike
+from qiskit_dynamics.arraylias import DYNAMICS_NUMPY_ALIAS as alias
```

Comment:
I think maybe we should call this `numpy_alias` instead of just `alias` now that there will be one for NumPy and one for SciPy.

Reply:
fixed at https://github.com/Qiskit-Extensions/qiskit-dynamics/pull/269/commits/79c45ceacadcaba54e2bb49824a34a19b4719ec8 .